<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 88, 87);
font: 57 24pt &quot;Ubuntu&quot;;
color:rgb(255,255, 255);</string>
   </property>
   <property name="text">
    <string>Nouvelle Annonce</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="villeEnt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateDebEnt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateFinEnt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="adr">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="prix">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ville">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;
</string>
   </property>
   <property name="text">
    <string>Ville</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateDeb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;</string>
   </property>
   <property name="text">
    <string>Date debut disponiblité</string>
   </property>
  </widget>
  <widget class="QLabel" name="datefin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;
</string>
   </property>
   <property name="text">
    <string>Date fin de disponibilité</string>
   </property>
  </widget>
  <widget class="QLabel" name="adress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;</string>
   </property>
   <property name="text">
    <string>Adresse retrait et retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;</string>
   </property>
   <property name="text">
    <string>Tarif</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Vemana2000&quot;;
</string>
   </property>
   <property name="text">
    <string>Photo (optionnelle)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrtiption</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajoutphoto">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 88, 87);</string>
   </property>
   <property name="text">
    <string>Ajouter photo</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="desc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 88, 87);
color:rgb(255,255, 255);</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 88, 87);
color:rgb(255,255, 255);</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
